pkg/grpc/server.go

```diff
@@ -193,10 +195,22 @@ func (a *apiImpl) Stop() bool {
 	a.listenersLock.Lock()
 	defer a.listenersLock.Unlock()
 
+	a.debugLog.Logf("Stopping %d listeners", len(a.listeners))
```

Comment:
Would that work for the non-test case? Isn't the `debugLog` nil in that case?

Reply:
The intention is to only print these lines in the test cases.